sites/next.skeleton.dev/src/examples/components/ratings/Example.svelte

```diff
@@ -0,0 +1,13 @@
+<script lang="ts">
+    import { Rating } from '@skeletonlabs/skeleton-svelte';
+	import Star from 'lucide-svelte/icons/star';
+</script>
+
+<Rating value={3} width="w-44">
+    {#snippet iconFull()}
+        <Star size={24} fill="white" />
```

Comment:
Remove the hardcoded white fill. Either use the default (per my other comment) or use a color pairing like `class="fill-surface-950-50"`. Currently this is impossible to see in light mode:

![Screenshot 2024-06-11 at 2 06 13 PM](https://github.com/skeletonlabs/skeleton/assets/1509726/e75a1f89-df46-4d55-b4af-00238ae679ee)

![Screenshot 2024-06-11 at 2 06 19 PM](https://github.com/skeletonlabs/skeleton/assets/1509726/87f943d6-9324-418f-b197-064d2ee53ee6)